telemetry/definitions/commonDefinitions.json

```diff
@@ -1432,6 +1432,11 @@
             ],
             "description": "The type of an SQS Queue"
         },
+        {
+            "name": "startUpComponent",
+            "type": "string",
+            "description": "The Toolkit component being initialized."
```

Comment:
there is already a "component" field https://github.com/aws/aws-toolkit-common/blob/b362bc7c5b65f12fee154b1c17ca896e5645cafc/telemetry/definitions/commonDefinitions.json#L819-L830

can it be re-used. if you need a free-form string, just cast it.

Reply:
defined a new "module" generic metadata to give us future reusability